Editor/Resources/EditorWindow/GameLiftPlugin.uss

```diff
@@ -0,0 +1,56 @@
+:root {
+    height: 100%;
+    width: 100%;
+    
+    --color-menu-background: #333333;
+    --color-content-background: #3F3F3F;
+    --color-button-primary: #2C5D87;
+    --color-button-hover: #386082;
+    
+    --font-size-menu: 18px;
+}
+
+.main {
+    flex-direction: row;
+    height: 100%;
+    width: 100%;
+}
+
+.main__menu {
+    width: 230px;
+    padding-top: 20px;
+    background-color: var(--color-menu-background);
+}
+
+.main__content {
+    flex-grow: 1;
+    background-color: var(--color-content-background);
+}
+
+.TabButton {
```

Comment:
we should be consistent with css class naming. On this file i see the following conventions
* TabButton (capitalization)
* main (capitalization)
* main__content (__)
* TabButton--selected (--)


Please pick one notation and ensure that all uss files stick to this.